frontend/src/pages/HomePage.jsx

```diff
@@ -3,30 +3,50 @@ import SearchBar from "../components/searchBar";
 import Template from "../components/template";
 import UploadNotesWindow from "../components/uploadNotesWindow";
 import ModalHeader from "../components/modalHeader";
-import { useState } from 'react';
-import "../css/homepage.css"
+import { useState } from "react";
+import "../css/homepage.css";
 
 function HomePage() {
-  const [buttonUpload, setButtonUpload] = useState(false);
-  const [updateSearch, setUpdateSearch] = useState(false);
+    const [buttonUpload, setButtonUpload] = useState(false);
+    const [updateSearch, setUpdateSearch] = useState(false);
 
-  return (
-    <>
-      <Template showSearch={false} showProfile={true} body={
-        <div className="d-flex flex-column center-home-content">
-          <ModalHeader title="Study Your Way" />
-          <div className="d-flex flex-column align-items-center">
-            <SearchBar showFilterBtn={false} updateSearch={updateSearch}/>
-            <div className="d-flex flex-row">
-              <Button title="UPLOAD" type="secondary" clicked={()=> setButtonUpload(true)} />
-              <Button title="SEARCH" type="primary" clicked={()=> setUpdateSearch(true)} />
-              <UploadNotesWindow trigger={buttonUpload} setTrigger={setButtonUpload}/>
+    return (
+        <>
+            <Template
+                showSearch={false}
+                showProfile={true}
+                body={
+                    <div className="d-flex flex-column center-home-content">
+                        <ModalHeader title="Study Your Way" />
+                        <div className="d-flex flex-column align-items-center">
+                            <SearchBar
+                                showFilterBtn={false}
+                                updateSearch={updateSearch}
+                            />
+                            <div className="d-flex flex-row">
+                                <Button
+                                    title="UPLOAD"
+                                    type="secondary"
+                                    clicked={() => setButtonUpload(true)}
+                                />
+                                <Button
+                                    title="SEARCH"
+                                    type="primary"
+                                    clicked={() => setUpdateSearch(true)}
+                                />
+                            </div>
+                        </div>
+                    </div>
+                }
+            />
+            <div>
```

Comment:
Why is this is a separate div?

Reply:
This is for blurring the entire background.